native/cocos/bindings/jswrapper/v8/debugger/inspector_socket_server.cpp

```diff
@@ -475,8 +480,18 @@ bool InspectorSocketServer::Start() {
     }
     state_ = ServerState::kRunning;
     // getaddrinfo sorts the addresses, so the first port is most relevant.
+    #if (CC_PLATFORM != CC_PLATFORM_NX)
     PrintDebuggerReadyMessage(host_, server_sockets_[0]->port(),
                               delegate_->GetTargetIds(), out_);
+    #else
+    auto nxHost = IpAddr();
+    auto ids    = delegate_->GetTargetIds();
+    for (const std::string &id : ids) {
+        SE_LOGD("Debugger listening..., visit [ devtools://devtools/bundled/js_app.html?v8only=true&ws=%s ] in chrome browser to debug!\n",
+                FormatWsAddress(nxHost, Port(), id, false).c_str());
+    }
+
+    #endif
```

Comment:
Why not just implement it in `PrintDebuggerReadyMessage()`? I think it is more elegant to implement it there.